shiptest.dme

```diff
@@ -744,8 +744,8 @@
 #include "code\datums\votes\_vote_datum.dm"
 #include "code\datums\votes\custom_vote.dm"
 #include "code\datums\votes\restart_vote.dm"
-#include "code\datums\weather\weather.dm"
 #include "code\datums\votes\transfer_vote.dm"
+#include "code\datums\weather\weather.dm"
```

Comment:
This is an irrelevant change. Swap this back so we don't have a DME edit tag here.